src/app/components/mnoe-users-list/mno-users-list.html

```diff
@@ -4,16 +4,16 @@
     <input type="text" ng-model="users.search" ng-change="searchChange()" ng-model-options='{ debounce: 1000 }' placeholder="{{'mnoe_admin_panel.dashboard.users.widget.list.search_users.placeholder.search_all' | translate}}" class="form-control input-sm search-bar" />
   </mno-widget-header>
   <mno-widget-body class="large no-padding">
-    <table class="table table-layout-fixed">
+    <table class="table table-layout-fixed" st-table="displayedUsers" st-safe-src="users.list">
```

Comment:
Can't you write:
```
<table class="table table-layout-fixed" st-table="users.list">
```

Reply:
I followed the official documentation

> If you are bringing in data asynchronously (from a remote database, restful endpoint, ajax call, etc.) you must use the stSafeSrc attribute. You must use a separate collection for both the base and safe collections or you may end up with an infinite loop.

Not using st-safe-src will empty the table after a sorting